packages/walletkit/src/bootnode-utils.ts

```diff
@@ -0,0 +1,30 @@
+import { NETWORK_NAME } from '../contracts/network-name'
+import GoogleStorageUtils from './google-storage-utils'
+import { Logger } from './logger'
+
+// The bucket where we store bootnode information for all the networks.
+const BootnodesGoogleStorageBucketName = `bootnodes`
+
+export default class BootnodeUtils {
```

Comment:
In Javascript/Typescript, it'd be more common to just leave these functions at the root scope instead of wrapping them in a class that has no data. Classes are more typically used when they're specifically needed (i.e. an encapsulation of both data and functionality), unlike in Java where everything is always a class.
Consider reshaping this :)